cmd/grafana-app-sdk/project_local.go

```diff
@@ -391,6 +393,9 @@ func generateKubernetesYAML(crdGenFunc func() (codejen.Files, error), pluginID s
 		// Prefix with "localhost/" to ensure that our local build uses our locally-built image
 		props.OperatorImage = fmt.Sprintf("localhost/%s", props.OperatorImage)
 	}
+	if config.GrafanaAnonymousAuth {
+		props.GrafanaAnonymousAuth = "Viewer"
```

Comment:
@stephaniehingtgen austin made this PR to make Viewer the default for app-sdk. fyi